parsl/providers/lsf/lsf.py

```diff
@@ -64,11 +72,19 @@ class LSFProvider(ClusterProvider, RepresentationMixin):
         :class:`~parsl.launchers.SrunLauncher`, or
         :class:`~parsl.launchers.AprunLauncher`
     move_files : Optional[Bool]: should files be moved? by default, Parsl will try to move files.
+    bsub_redirection: Bool
+        Should a redirection symbol "<" be included when submitting jobs, i.e., Bsub < job_script.
```

Comment:
for consideration later - not affecting this review - do most LSF systems what `<` or not?
If LSF generally wants `<`, then the default should be to include `<`.